preview-src/ui-model.yml

```diff
@@ -68,7 +68,7 @@ page:
   version: 'dev'
   displayVersion: 'dev'
   module: ROOT
-  editUrl: https://example.com/project-bonita/blob/master/index.adoc
+  editUrl: https://example.com/project-bonita/edit/master/index.adoc
```

Comment:
ℹ️ looks fine, the github url are something like https://github.com/bonitasoft/bonita-doc/edit/2022.2/modules/ROOT/pages/index.adoc